### PR TITLE
Travis and build script for maxmin

### PR DESCRIPTION
**Pull request by ezeeyahoo**
Hi,

Please review both .yml and build script.

Thanks

### CODE REVIEW

File: m/maxmin/maxmin_ubuntu_18.04.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#! /bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Package	: maxmin
+# Version	: 
+# Source repo	: https://github.com/gaearon/react-hot-loader
+# Tested on	: ubuntu_18.04
+# Script License: MIT License
```
Comment (seth-priya) [inline m/maxmin/maxmin_ubuntu_18.04.sh:8]: Script should always be published under the Apache 2.0 license

File: travis-ymls/maxmin.travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+language: node_js
```
Comment (seth-priya) [inline travis-ymls/maxmin.travis.yml:1]: the header for the travis.yml file is missing, please refer to https://github.com/ppc64le/build-scripts/blob/master/travis-ymls/README.md

File: travis-ymls/maxmin.travis.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+arch:
+  - amd64
+  - ppc64le
+node_js:
+  - '14'
+#   - '12'
```
Comment (seth-priya) [inline travis-ymls/maxmin.travis.yml:7]: are the builds for these versions failing?

Comment (ezeeyahoo) [reply travis-ymls/maxmin.travis.yml]: fixed typo

File: travis-ymls/maxmin.travis.yml (hunk 2 above)
Comment (ezeeyahoo) [reply travis-ymls/maxmin.travis.yml:11]: added

File: m/maxmin/maxmin_ubuntu_18.04.sh (hunk 1 above)
Comment (ezeeyahoo) [reply m/maxmin/maxmin_ubuntu_18.04.sh]: fixed

**Comment by ezeeyahoo** [comment]
why is commit 7338cf3adf2264a1f12b277b492bf434ed40ca36 showing in my branch?

**Comment by seth-priya** [comment]
@ezeeyahoo did you do a git pull before pushing your changes?

**Comment by seth-priya** [comment]
No response, closing

**Pull request closed without merging**